.rubocop.yml

```diff
@@ -2,6 +2,8 @@ AllCops:
   TargetRubyVersion: 2.2
   Exclude:
     - 'vendor/**/*'
+    - 'db/schema.rb'
+    - 'db/migrate/**/*'
```

Comment:
ignore all `ActiveRecord` schema and migration updates as they are handled automatically.